Assets/Scripts/PinsController.cs

```diff
@@ -1,114 +0,0 @@
-﻿using System.Collections.Generic;
-using UnityEngine;
-
-
-public class PinsController : MonoBehaviour
-{
-
-	public Pin PinPrefab;
-	private PinPosition[] pinPositions;
-	private List<Pin> pins;
-	private Animator anim;
-	private BowlingBall ball;
-	
-	
-	private void Start ()
-	{
-		ball = FindObjectOfType<BowlingBall>();
-		pinPositions = GetComponentsInChildren <PinPosition>();
-		pins = new List<Pin>();
-		anim = GetComponent<Animator>();
-	
-		SpawnPins();
-		LowerPins();
-	}
-
-	public void DebugStrike()
-	{
-		foreach (var pin in pins)
-		{
-			if (pin != null)
-			{
-				pin.GetComponent<Rigidbody>().AddForce(Vector3.right * 200, ForceMode.Impulse);
```

Comment:
Nice, I like that you've added a debug function for creating kinds of rolls. Good that your using Impulse mode force.